---
title: MSSQLSERVER_26013 | Документация Майкрософт
ms.custom: ''
ms.date: 04/04/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: supportability
ms.topic: language-reference
helpviewer_keywords:
- 26013 (Database Engine error)
ms.assetid: 7362fca4-b005-4872-bdfa-a3e32f834db8
author: MashaMSFT
ms.author: mathoma
manager: craigg
ms.openlocfilehash: 560d923e42ae5516fb64727236fd5730d8e7694d
ms.sourcegitcommit: 61381ef939415fe019285def9450d7583df1fed0
ms.translationtype: HT
ms.contentlocale: ru-RU
ms.lasthandoff: 10/01/2018
ms.locfileid: "47638632"
---
# <a name="mssqlserver26013"></a>MSSQLSERVER_26013
[!INCLUDE[appliesto-ss-xxxx-xxxx-xxx-md](../../includes/appliesto-ss-xxxx-xxxx-xxx-md.md)]
  
## <a name="details"></a>Сведения  
  
|||  
|-|-|  
|Название продукта|SQL Server|  
|Идентификатор события|26013|  
|Источник события|MSSQLSERVER|  
|Компонент|SQLEngine|  
|Символическое имя|SNI_SSL_READY|  
|Текст сообщения|Сертификат [Cert Hash(sha1) "%hs"] был успешно загружен для шифрования.|  
  
## <a name="explanation"></a>Объяснение  
Это информационное сообщение. Указанный сертификат был успешно загружен в [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] и может использоваться для шифрования.  
  
## <a name="user-action"></a>Действие пользователя  
Никаких действий не требуется.  
  
